templates/front/my_checks_desktop.html

```diff
@@ -12,6 +12,7 @@
         <th></th>
     </tr>
     {% for check in checks %}
+    {% if check.id in allowed_checks or check.user_id == user.id %}
```

Comment:
if `check.user_id == user.id` should come first. We should figure out a way to optimize the extra lookup here for the second condition `check.id in allowed_checks`.
Is there a reason for not looping through the intersect of checks and allow_checks here?